lib/biran/config.rb

```diff
@@ -70,6 +71,10 @@ def bindings
       @bindings ||= %i[db_config]
     end
 
+    def extra_config_suffix
+      @extra_config_suffix ||= 'extras'.freeze
```

Comment:
instead of freezing this, add a top comment like: `# frozen_string_literal: true`. this will indicate the compiler to freeze all string literals

Reply:
Should I do that for all the things listed like this? What about the things like `%i[app].freeze`? Remove freeze on those two?

Reply:
that is not a literal string, it is an array of atoms, so those are fine frozen manually